cedar-policy-validator/src/typecheck.rs

```diff
@@ -1174,6 +1174,11 @@ impl<'a> SingleEnvTypechecker<'a> {
                     },
                 )
             }
+            // PANIC SAFETY: We should never be trying to type check or evaluate code that allows errors - This is avoided by using the default cst-to-ast parsing which fails on parse errors
```

Comment:
Split the comment into at least two lines.

Reply:
Will do